dll/win32/shell32/shlexec.cpp

```diff
@@ -1558,8 +1558,7 @@ static HRESULT ShellExecute_ContextMenuVerb(LPSHELLEXECUTEINFOW sei)
     __SHCloneStrWtoA(&parameters, sei->lpParameters);
 
     BOOL fDefault = !sei->lpVerb || !sei->lpVerb[0];
-    CMINVOKECOMMANDINFOEX ici = { sizeof(ici) };
-    ici.fMask = (sei->fMask & (SEE_MASK_NO_CONSOLE | SEE_MASK_ASYNCOK | SEE_MASK_FLAG_NO_UI)) | CMIC_MASK_UNICODE;
+    CMINVOKECOMMANDINFOEX ici = { sizeof(ici), SeeFlagsToCmicFlags(sei->fMask) | CMIC_MASK_UNICODE };
```

Comment:
Why not? Brace initialize means fewer leftover bytes to memset.

Reply:
The line is less than 100 characters but I can change it back if you feel that strongly about it.

Reply:
i prefer the separate line indeed because, contrary to the struct size where it's common that it's the first field, it's not obvious that the flags field is the 2nd.

Reply:
If a struct has size and flags, flags is always number 2 but OK.